src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -117,22 +116,6 @@ export default class StyleConfigurator {
     });
   }
 
-  // TODO: to be removed as it will be configured in configureGatewaysStyle
-  // left just to not break current rendering
-  private configureParallelGatewayStyle(): void {
-    const style = this.cloneDefaultVertexStyle();
-    style[this.mxConstants.STYLE_SHAPE] = this.mxConstants.SHAPE_RHOMBUS;
-    style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.RhombusPerimeter;
-    style[this.mxConstants.STYLE_VERTICAL_ALIGN] = 'top';
-    style[this.mxConstants.STYLE_STROKECOLOR] = '#96A826';
-    style[this.mxConstants.STYLE_STROKEWIDTH] = 1.7;
-
-    style[this.mxConstants.STYLE_SPACING_TOP] = 55;
-    style[this.mxConstants.STYLE_SPACING_RIGHT] = 110;
-    style[this.mxConstants.STYLE_GRADIENTCOLOR] = '#E9ECB1';
-    this.putCellStyle(ShapeBpmnElementKind.GATEWAY_PARALLEL, style);
-  }
-
   private configureGatewaysStyle(): void {
```

Comment:
@aibcmars Can you remove unnecessary line `style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.RhombusPerimeter;` ?

Reply:
can we please stop noticing this? we have to understand why this is put in all mxGraph examples
This seems to be ok without for our rendering but we do not know what is the purpose of this.
I suggest we manage this later and track this with a dedicated issue

What I have understood so far is that the perimeter provides the list of points that can be used to attach edges to the current shape. This could be usefull for automatic rendering.
I know we are supposed to have the waypoints in the BPMN files that should define the point on the shape where the edge must be attached, but without exactly knowing what is going on, it is to early to remove the perimeters from the style IMHO

Reply:
I agree with Thomas, as yesterday stated: we have found it occurs in all mxgraph examples concerning RHOMBUS.